src/System.Private.ServiceModel/tests/System.Private.ServiceModel.Tests.csproj

```diff
@@ -14,7 +14,7 @@
     <ProjectGuid>{BB3B114D-68D7-4E36-9EBE-127E694E6EC9}</ProjectGuid>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs" Exclude="Scenarios\SelfHostWcfService\*" />
+    <Compile Include="**\*.cs" Exclude="Scenarios\**\*" />
```

Comment:
That include was pulling all the scenario tests into the unit tests for S.P.SM -- it's been a nuisance bug I finally decided to fix.  Before this change, an F5 on this unit test project attempts to run all the scenario tests too, and they fail.  After the change, F5 runs only the unit tests under this folder.  Issue #75 is responsible for refactoring the folder structure to separate the unit tests from the scenario tests better.